README.md

```diff
@@ -48,3 +51,10 @@ Resource Reservation Instructions (Optional)
 7. On each slave with the new settings, start the mesos slave by running
 <br>`sudo service mesos-slave start`</br>.
 
+Shutdown Instructions (Optional)
+--------------------------
+
+1. In Marathon (or your other long-running process monitor) stop the hdfs scheduler application
+2. Shutdown the hdfs framework in Mesos: `curl -d "frameworkId=YOUR_FRAMEWORK_ID" -X POST http://YOUR_MESOS_URL:5050/master/shutdown`
+3. Access your zookeeper instance: `/PATH/TO/zookeeper/bin/zkCli.sh`
+4. Remove hdfs-mesos framework state from zookeeper: `rmr /hdfs-mesos`
```

Comment:
Let's also add an explanation of this property, mesos.hdfs.native-hadoop-binaries in the readme.